spec/rspec/support/differ_spec.rb

```diff
@@ -261,6 +261,35 @@ def inspect; "<BrokenObject>"; end
           expect(diff).to be_diffed_as(expected_diff)
         end
 
+      context 'when special-case objects inside hashes' do
```

Comment:
This doc string doesn't really make sense grammatically.  Should it be "with special-case objects inside hashes" or "when special-case objects are inside hashes"?